Fitness.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 48;
```

Comment:
Why is this going down? What is this?

Reply:
Not sure--I'll look into it!

Reply:
It refers the the version of xcode the project is compatible with--I imagine it changed since some of us have different versions. I changed it back to 51 with no issues!